src/Webapi/Webapi__Navigator.res

```diff
@@ -0,0 +1,48 @@
+type t
+
+/* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorid */
+
+@get external userAgent: t => string = "userAgent"
+@get external vendor: t => string = "vendor"
+
+/* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorlanguage */
+
+/**
+ * Returns a BCP 47 language tag
+ */
+@get external language: t => string = "language"
+
+/**
+ * Experimental. Returns a readonly array of BCP 47 language tags
+ */
+// @get external languages: t => array<string> = "languages"
+
+/* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatoronline */
+
+@get external onLine: t => bool = "onLine"
+
+/* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorcontentutils */
+
+@send
+external _registerProtocolHandler: (t, string, string) => unit = "registerProtocolHandler"
+
+let registerProtocolHandler = (t, ~scheme, ~url) => _registerProtocolHandler(t, scheme, url)
```

Comment:
@TheSpyder I wrapped this, think you mentioned on another PR that having multiple arguments of the same time in a row was confusing and I agree.

Reply:
yes, thank you 👍 

You don't need to wrap it, though, externals support named arguments just as well as any other function :)

`@send external registerProtocolHandler: (t, ~scheme:string, ~url:string) => unit = "registerProtocolHandler"`

Reply:
Ah cool, I thought for some reason it compiled to an object as an arg with the named arguments

Reply:
Also, when we do wrap I prefer to shadow. Otherwise the original method leaks out into the module namespace and I find it confusing to see both in autocomplete.

Reply:
> Ah cool, I thought for some reason it compiled to an object as an arg with the named arguments

This is a specific technique used in some places with `@obj` when an external takes an object param instead of multiple arguments. A subtle difference in ReScript code, but very different JS generated :)